### ISSUE TITLE
Cannot go get -u om because of submoduled jhanda?!?

### ISSUE THREAD

**Issue by camelpunch** [pivotal-cf/om#108]
```
$ go get -u github.com/pivotal-cf/om

package github.com/pivotal-cf/om/vendor/github.com/pivotal-cf/jhanda/commands: /Users/pivotal/go/src/github.com/pivotal-cf/om/vendor/github.com/pivotal-cf/jhanda/.git exists but is not a directory
```

**Comment by joshzarrabi** [comment]
This looks like an issue with `go get`, because we put our submodules in the `vendor` directory which apparently doesn't work. https://github.com/golang/go/issues/17522. It seems like they are working on it based on the link below though.

**Comment by zachgersh** [comment]
@joshzarrabi I've actually fixed this with my PR, submodule strategy was not how we had vendored code in the past.

**Comment by acrmp** [comment]
@camelpunch Thanks for reporting this! We've merged Zach's pull request (#109) which fixes this.

Andrew & Riz
